lib/talon/resource.ex

```diff
@@ -167,7 +176,7 @@ defmodule Talon.Resource do
       def resource_paths(%{talon: talon} = _talon) do
         talon.resources()
         |> Enum.map(fn talon_resource ->
-          schema = talon_resource.schema()
+          schema = talon_resource.schema() # TODO: let resource determine presentation name (DJS)
```

Comment:
I believe your comment is more applicable to the line below. The `Talon.Utils.titleize(schema)` should be changed to be an overridable function call.